<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>870</width>
    <height>515</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MNI_P2</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>Configurações</string>
       </attribute>
       <widget class="QLabel" name="label">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>171</width>
          <height>17</height>
         </rect>
        </property>
        <property name="text">
         <string>Quantidade de Círculos:</string>
        </property>
       </widget>
       <widget class="QSpinBox" name="spinBox_QtdC">
        <property name="geometry">
         <rect>
          <x>170</x>
          <y>0</y>
          <width>71</width>
          <height>27</height>
         </rect>
        </property>
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="value">
         <number>3</number>
        </property>
       </widget>
       <widget class="QTableWidget" name="tableC">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>50</y>
          <width>551</width>
          <height>411</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_2">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>30</y>
          <width>67</width>
          <height>17</height>
         </rect>
        </property>
        <property name="text">
         <string>Círculos:</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_3">
        <property name="geometry">
         <rect>
          <x>690</x>
          <y>30</y>
          <width>67</width>
          <height>17</height>
         </rect>
        </property>
        <property name="text">
         <string>D:</string>
        </property>
       </widget>
       <widget class="QTableWidget" name="tableD">
        <property name="geometry">
         <rect>
          <x>685</x>
          <y>50</y>
          <width>151</width>
          <height>411</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_4">
        <property name="geometry">
         <rect>
          <x>560</x>
          <y>126</y>
          <width>101</width>
          <height>31</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>34</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>* R =</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton">
        <property name="geometry">
         <rect>
          <x>560</x>
          <y>200</y>
          <width>99</width>
          <height>27</height>
         </rect>
        </property>
        <property name="text">
         <string>Calcular</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>Gauss</string>
       </attribute>
       <widget class="QLabel" name="label_5">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>181</width>
          <height>17</height>
         </rect>
        </property>
        <property name="text">
         <string>Resultado Usando Gauss:</string>
        </property>
       </widget>
       <widget class="QRadioButton" name="radioButtonGauss">
        <property name="geometry">
         <rect>
          <x>680</x>
          <y>0</y>
          <width>161</width>
          <height>22</height>
         </rect>
        </property>
        <property name="text">
         <string>Usar Pivoteamento</string>
        </property>
       </widget>
       <widget class="QTableWidget" name="tableRGauss">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>40</y>
          <width>121</width>
          <height>331</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_9">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>380</y>
          <width>81</width>
          <height>17</height>
         </rect>
        </property>
        <property name="text">
         <string>Tempo:</string>
        </property>
       </widget>
       <widget class="QLabel" name="labelTimeGauss">
        <property name="geometry">
         <rect>
          <x>90</x>
          <y>380</y>
          <width>621</width>
          <height>17</height>
         </rect>
        </property>
        <property name="text">
         <string>0</string>
        </property>
       </widget>
       <widget class="QLabel" name="labelInterationsGauss">
        <property name="geometry">
         <rect>
          <x>90</x>
          <y>410</y>
          <width>621</width>
          <height>17</height>
         </rect>
        </property>
        <property name="text">
         <string>0</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_12">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>410</y>
          <width>81</width>
          <height>17</height>
         </rect>
        </property>
        <property name="text">
         <string>Iterações:</string>
        </property>
       </widget>
       <widget class="QLabel" name="labelErrorGauss">
        <property name="geometry">
         <rect>
          <x>90</x>
          <y>440</y>
          <width>621</width>
          <height>17</height>
         </rect>
        </property>
        <property name="text">
         <string>0</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_14">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>440</y>
          <width>91</width>
          <height>17</height>
         </rect>
        </property>
        <property name="text">
         <string>Erro residual:</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_21">
        <property name="geometry">
         <rect>
          <x>370</x>
          <y>0</y>
          <width>101</width>
          <height>17</height>
         </rect>
        </property>
        <property name="text">
         <string>Passo a Passo:</string>
        </property>
       </widget>
       <widget class="QPlainTextEdit" name="plainTextEditGauss">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="geometry">
         <rect>
          <x>370</x>
          <y>20</y>
          <width>471</width>
          <height>351</height>
         </rect>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
       <widget class="QLabel" name="label_23">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>20</y>
          <width>67</width>
          <height>17</height>
         </rect>
        </property>
        <property name="text">
         <string>Raios:</string>
        </property>
       </widget>
       <widget class="QTableWidget" name="tableAreaGauss">
        <property name="geometry">
         <rect>
          <x>190</x>
          <y>40</y>
          <width>121</width>
          <height>331</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_24">
        <property name="geometry">
         <rect>
          <x>190</x>
          <y>20</y>
          <width>67</width>
          <height>17</height>
         </rect>
        </property>
        <property name="text">
         <string>Áreas:</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_3">
       <attribute name="title">
        <string>Gauss-Jordan</string>
       </attribute>
       <widget class="QTableWidget" name="tableRGaussJordan">
        <property name="geometry">
         <rect>
          <x>4</x>
          <y>40</y>
          <width>121</width>
          <height>331</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_6">
        <property name="geometry">
         <rect>
          <x>4</x>
          <y>0</y>
          <width>231</width>
          <height>17</height>
         </rect>
        </property>
        <property name="text">
         <string>Resultado Usando Gauss-Jordan:</string>
        </property>
       </widget>
       <widget class="QRadioButton" name="radioButtonGaussJordan">
        <property name="geometry">
         <rect>
          <x>680</x>
          <y>0</y>
          <width>161</width>
          <height>22</height>
         </rect>
        </property>
        <property name="text">
         <string>Usar Pivoteamento</string>
        </property>
       </widget>
       <widget class="QLabel" name="labelTimeGaussJordan">
        <property name="geometry">
         <rect>
          <x>90</x>
          <y>380</y>
          <width>621</width>
          <height>17</height>
         </rect>
        </property>
        <property name="text">
         <string>0</string>
        </property>
       </widget>
       <widget class="QLabel" name="labelErrorGaussJordan">
        <property name="geometry">
         <rect>
          <x>90</x>
          <y>440</y>
          <width>621</width>
          <height>17</height>
         </rect>
        </property>
        <property name="text">
         <string>0</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_13">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>410</y>
          <width>81</width>
          <height>17</height>
         </rect>
        </property>
        <property name="text">
         <string>Iterações:</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_15">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>440</y>
          <width>91</width>
          <height>17</height>
         </rect>
        </property>
        <property name="text">
         <string>Erro residual:</string>
        </property>
       </widget>
       <widget class="QLabel" name="labelInterationsGaussJordan">
        <property name="geometry">
         <rect>
          <x>90</x>
          <y>410</y>
          <width>621</width>
          <height>17</height>
         </rect>
        </property>
        <property name="text">
         <string>0</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_10">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>380</y>
          <width>81</width>
          <height>17</height>
         </rect>
        </property>
        <property name="text">
         <string>Tempo:</string>
        </property>
       </widget>
       <widget class="QPlainTextEdit" name="plainTextEditGaussJordan">
        <property name="geometry">
         <rect>
          <x>370</x>
          <y>20</y>
          <width>471</width>
          <height>351</height>
         </rect>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
       <widget class="QLabel" name="label_22">
        <property name="geometry">
         <rect>
          <x>370</x>
          <y>0</y>
          <width>101</width>
          <height>17</height>
         </rect>
        </property>
        <property name="text">
         <string>Passo a Passo:</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_25">
        <property name="geometry">
         <rect>
          <x>190</x>
          <y>20</y>
          <width>67</width>
          <height>17</height>
         </rect>
        </property>
        <property name="text">
         <string>Áreas:</string>
        </property>
       </widget>
       <widget class="QTableWidget" name="tableAreaGaussJordan">
        <property name="geometry">
         <rect>
          <x>190</x>
          <y>40</y>
          <width>121</width>
          <height>331</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_26">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>20</y>
          <width>67</width>
          <height>17</height>
         </rect>
        </property>
        <property name="text">
         <string>Raios:</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_4">
       <attribute name="title">
        <string>Comparativo</string>
       </attribute>
       <widget class="QRadioButton" name="radioButtonComp">
        <property name="geometry">
         <rect>
          <x>680</x>
          <y>0</y>
          <width>161</width>
          <height>22</height>
         </rect>
        </property>
        <property name="text">
         <string>Usar Pivoteamento</string>
        </property>
       </widget>
       <widget class="QTableWidget" name="tableRGaussComp">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>40</y>
          <width>121</width>
          <height>331</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_7">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>181</width>
          <height>17</height>
         </rect>
        </property>
        <property name="text">
         <string>Resultado Usando Gauss:</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_8">
        <property name="geometry">
         <rect>
          <x>420</x>
          <y>0</y>
          <width>231</width>
          <height>17</height>
         </rect>
        </property>
        <property name="text">
         <string>Resultado Usando Gauss-Jordan:</string>
        </property>
       </widget>
       <widget class="QTableWidget" name="tableRGaussJordanComp">
        <property name="geometry">
         <rect>
          <x>500</x>
          <y>40</y>
          <width>121</width>
          <height>331</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="labelTimeGaussComp">
        <property name="geometry">
         <rect>
          <x>90</x>
          <y>380</y>
          <width>401</width>
          <height>17</height>
         </rect>
        </property>
        <property name="text">
         <string>0</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_16">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>410</y>
          <width>81</width>
          <height>17</height>
         </rect>
        </property>
        <property name="text">
         <string>Iterações:</string>
        </property>
       </widget>
       <widget class="QLabel" name="labelErrorGaussComp">
        <property name="geometry">
         <rect>
          <x>90</x>
          <y>440</y>
          <width>401</width>
          <height>17</height>
         </rect>
        </property>
        <property name="text">
         <string>0</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_17">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>440</y>
          <width>91</width>
          <height>17</height>
         </rect>
        </property>
        <property name="text">
         <string>Erro residual:</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_11">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>380</y>
          <width>81</width>
          <height>17</height>
         </rect>
        </property>
        <property name="text">
         <string>Tempo:</string>
        </property>
       </widget>
       <widget class="QLabel" name="labelInterationsGaussComp">
        <property name="geometry">
         <rect>
          <x>90</x>
          <y>410</y>
          <width>401</width>
          <height>17</height>
         </rect>
        </property>
        <property name="text">
         <string>0</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_18">
        <property name="geometry">
         <rect>
          <x>500</x>
          <y>380</y>
          <width>81</width>
          <height>17</height>
         </rect>
        </property>
        <property name="text">
         <string>Tempo:</string>
        </property>
       </widget>
       <widget class="QLabel" name="labelInterationsGaussJordanComp">
        <property name="geometry">
         <rect>
          <x>590</x>
          <y>410</y>
          <width>251</width>
          <height>17</height>
         </rect>
        </property>
        <property name="text">
         <string>0</string>
        </property>
       </widget>
       <widget class="QLabel" name="labelTimeGaussJordanComp">
        <property name="geometry">
         <rect>
          <x>590</x>
          <y>380</y>
          <width>251</width>
          <height>17</height>
         </rect>
        </property>
        <property name="text">
         <string>0</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_19">
        <property name="geometry">
         <rect>
          <x>500</x>
          <y>440</y>
          <width>91</width>
          <height>17</height>
         </rect>
        </property>
        <property name="text">
         <string>Erro residual:</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_20">
        <property name="geometry">
         <rect>
          <x>500</x>
          <y>410</y>
          <width>81</width>
          <height>17</height>
         </rect>
        </property>
        <property name="text">
         <string>Iterações:</string>
        </property>
       </widget>
       <widget class="QLabel" name="labelErrorGaussJordanComp">
        <property name="geometry">
         <rect>
          <x>590</x>
          <y>440</y>
          <width>251</width>
          <height>17</height>
         </rect>
        </property>
        <property name="text">
         <string>0</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_27">
        <property name="geometry">
         <rect>
          <x>690</x>
          <y>20</y>
          <width>67</width>
          <height>17</height>
         </rect>
        </property>
        <property name="text">
         <string>Áreas:</string>
        </property>
       </widget>
       <widget class="QTableWidget" name="tableAreaGaussJordanComp">
        <property name="geometry">
         <rect>
          <x>690</x>
          <y>40</y>
          <width>121</width>
          <height>331</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_28">
        <property name="geometry">
         <rect>
          <x>180</x>
          <y>20</y>
          <width>67</width>
          <height>17</height>
         </rect>
        </property>
        <property name="text">
         <string>Áreas:</string>
        </property>
       </widget>
       <widget class="QTableWidget" name="tableAreaGaussComp">
        <property name="geometry">
         <rect>
          <x>180</x>
          <y>40</y>
          <width>121</width>
          <height>331</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_29">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>20</y>
          <width>67</width>
          <height>17</height>
         </rect>
        </property>
        <property name="text">
         <string>Raios:</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_30">
        <property name="geometry">
         <rect>
          <x>500</x>
          <y>20</y>
          <width>67</width>
          <height>17</height>
         </rect>
        </property>
        <property name="text">
         <string>Raios:</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_5">
       <attribute name="title">
        <string>Analise das Circunferencias</string>
       </attribute>
       <widget class="QGraphicsView" name="graphicsView">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>851</width>
          <height>461</height>
         </rect>
        </property>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
